releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,34 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+
+## 1.0.0 - March 9, 2023
```

Comment:
We should list all differences in every stable release compared with the previous one. Since this is our first stable release, we should list all key features supported in this release, even though some have been mentioned in preview releases.

Reply:
Added a `Features` section.